From 49ece949ace07186b0d5f4528dfe924d18a441c1 Mon Sep 17 00:00:00 2001
From: Peter Vary <pvary@cloudera.com>
Date: Mon, 29 May 2017 13:08:12 +0200
Subject: [PATCH 1153/1164] CLOUDERA-BUILD: CDH-54490 hive.server2.enable.SSL
 deprecation handling test fix

Change-Id: I2e5e787b12f9b40a06f22992032cd19afc4063d5
(cherry picked from commit ee4f1b22a7632beda6e596a541efafa0c81e3e7f)
---
 .../test/java/org/apache/hive/jdbc/TestSSL.java    |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestSSL.java b/itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestSSL.java
index ef1f12c..993c94c 100644
--- a/itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestSSL.java
+++ b/itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestSSL.java
@@ -498,12 +498,12 @@ private void setBinaryConfOverlay(Map<String, String> confOverlay) {
    * Test SSL client connection to SSL server
    * @throws Exception
    */
-  // FIXME: Failing since 5.11. Have to check why.
+  @Test
   public void testSSLDeprecatConfig() throws Exception {
     setSslConfOverlay(confOverlay);
     // Test in binary mode
     setBinaryConfOverlay(confOverlay);
-    clearSslConfOverlay(confOverlay);
+    confOverlay.remove(ConfVars.HIVE_SERVER2_USE_SSL.varname);
     confOverlay.put("hive.server2.enable.SSL", "true");
     // Start HS2 with SSL
     miniHS2.start(confOverlay);
-- 
1.7.9.5

